### PR TITLE
Adding MATFeaturizer to docs

### PR DESCRIPTION
**Pull request by atreyamaj**
# Pull Request Template

## Type of change

Please check the option that is related to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - In this case, we recommend to discuss your modification on GitHub issues before creating the PR
- [x] Documentations (modification for documents)

## Checklist

- [ ] My code follows [the style guidelines of this project](https://deepchem.readthedocs.io/en/latest/development_guide/coding.html)
  - [ ] Run `yapf -i <modified file>` and check no errors (**yapf version must be  0.22.0**)
  - [ ] Run `mypy -p deepchem` and check no errors
  - [ ] Run `flake8 <modified file> --count` and check no errors
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New unit tests pass locally with my changes
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by atreyamaj** [comment]
The MATFeaturizer docstring is now being rendered.

Screenshot from 2021-07-14 23-34-53

Screenshot from 2021-07-14 23-35-00

File: deepchem/models/torch_models/layers.py [hunk 1]
```diff
@@ -0,0 +1,514 @@
+import math
```
Comment (rbharath) [inline deepchem/models/torch_models/layers.py:1]: This file shouldn't be in the current PR

**Review by rbharath — COMMENTED** [review]
I think you have an extra file in the PR. Once that's out we can merge in

File: deepchem/models/torch_models/layers.py (hunk 1 above)
Comment (atreyamaj) [reply deepchem/models/torch_models/layers.py:1]: Oh looks like that got added in by mistake. I'll close this PR and open up a fresh one.

**Pull request closed without merging**